buildkite/setup-windows.ps1

```diff
@@ -68,6 +68,11 @@ Write-Host "Installing curl..."
 # FYI: choco installs curl.exe in C:\ProgramData\chocolatey\bin (which is on the PATH).
 & choco install curl
 
+## Install LLVM/Clang
+Write-Host "Installing llvm..."
+# FYI: choco installs clang in C:\Program Files\LLVM\bin (which is not on the PATH).

```

Comment:
Don't we have to put it on the PATH then? Or how can Bazel find it? :)

Reply:
Bazel looks at the registry to find Clang, we can also set `BAZEL_LLVM` here, I'm adding it.